docs/modules/publisher-commandline.md

```diff
@@ -527,6 +535,10 @@ The `Alternative` field, where present, refers to the CLI argument applicable in
                                    can not be sent quick enough to IoTHub
                                    Min: 1024
                                    Default: 8192
+          --mc, --monitoreditemdatachangetrigger=VALUE
```

Comment:
Was this command line option part of OPC Publisher 2.5?

Reply:
ups, my bad, wrong section! I'll update